src/Base58Check.mo

```diff
@@ -0,0 +1,25 @@
+import Text "mo:base/Text";
+import Nat8 "mo:base/Nat8";
+import Array "mo:base/Array";
+import Iter "mo:base/Iter";
+import SHA256 "../motoko-sha/src/SHA256";
+import Base58 "./Base58";
+
+module {
+  public func encode(input : [Nat8]) : Text {
+    // Add 4-byte hash check to the end.
+    let hash : [Nat8] = SHA256.sha256(SHA256.sha256(input));
+    let vch : [var Nat8] = Array.init<Nat8>(input.size() + 4, 0);
```

Comment:
nit: ditto regarding abbreviations. I don't understand what `vch` stands for.

Reply:
updated